frontend/src/PagesAdmin/RecruitmentPositionFormAdminPage/RecruitmentPositionForm.tsx

```diff
@@ -0,0 +1,265 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import type React from 'react';
+import { useEffect } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { z } from 'zod';
+import {
+  Button,
+  Checkbox,
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  Input,
+  Textarea,
+} from '~/Components';
+import { postRecruitmentPosition, putRecruitmentPosition } from '~/api';
+import { KEY } from '~/i18n/constants';
+import { reverse } from '~/named-urls';
+import { ROUTES } from '~/routes';
+import { NON_EMPTY_STRING } from '~/schema/strings';
+import styles from './RecruitmentPositionFormAdminPage.module.scss';
+
+export const recruitmentPositionSchema = z.object({
```

Comment:
Kortere variabelnavn, og trenger ikke eksportere

```suggestion
const schema = z.object({
```